src/components/user/tour/tour.js

```diff
@@ -1,26 +1,46 @@
 import React from "react";
+import Button from "@material-ui/core/Button";
 import { DataGrid } from '@material-ui/data-grid';
 import { TOUR_ROWS, TOUR_COLUMNS } from './utils/constants';
+import CreateTour from "./create-tour";
+import AddIcon from '@material-ui/icons/Add';
+import "./tour.css"
 
 export default class Tour extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
-      tourData: []
+      tourData: [],
+      isCreateModalOpen: false
     };
+    this.handleCreateModalToggle = this.handleCreateModalToggle.bind(this);
   }
 
   getData(){
     this.setState({tourData: TOUR_ROWS});
   };
 
+  handleCreateModalToggle() {
+    this.setState((state) => ({
```

Comment:
You can use the current value of the state without passing the function to `setState`.

```
this.setState({
  isCreateModalOpen: !this.state.isCreateModalOpen
})
```